### ISSUE TITLE
Gangut is not an FBB

### ISSUE THREAD

**Issue by Accelyte** [KC3Kai/KC3Kai#2029]
![chrome_2017-05-05_23-43-23](https://cloud.githubusercontent.com/assets/21363469/25766799/a7e7b072-31ec-11e7-8c97-73be84422b2a.png)
Wrong DB?

**Comment by Vultren** [comment]
This is not wrong. Gangut has a ship classification of 8 which is Fast Battleship even though her speed is slow.
https://vignette1.wikia.nocookie.net/kancolle/images/b/b3/EJwNwlEOgyAMANC7cADaIjLxNgQJmmlLoMaPZXff8t7H3P00q9lV21gBtmNk6ZsdKj3VYqtIPUtqx7BZLkiqKe9XYR1Ar-AwRAzkCf2yTA4mjMHHGB3S_Ec4w81vlodt42q-PwXHIs4.png/revision/latest?cb=20170505074150

**Comment by Accelyte** [comment]
So what would happen in fleet routings that require fast fleets? Would she count as an FBB or not?

**Comment by Vultren** [comment]
From the wiki:

> While all of Gangut's variants are considered "Fast Battleships" under branching rules, her base speed is "Slow". This means that while Gangut can be used in fleet compositions requiring Fast Battleships (such as in the Escort Fleet of a Combined Fleet), her slow speed will make her ineligible for routes requiring a 'Fast' fleet.

**Comment by Vultren** [comment]
Speed and Classification are pretty much independent from each other. Another way to put it is Gangut is more like a battlecruiser like the Kongo-class so that puts her in the FBB class. However IIRC she's slower than Kaga historically (24.1 knots compared to 28), so that puts her in the slow speed.

**Comment by sinsinpub** [comment]
u should send this ticket to Tanaka 😂

**Comment by Accelyte** [comment]
So she can be used in a the fleet comp but will still cause off-routing?
Personally I think a label should be added in the KC3 library just to clarify, because not everyone is going to look it up.

**Comment by c933103** [comment]
Nope, it depend on what the route actually require. For instance, if a
branching rule require battleships, and then require fast fleet, then
Gangut would not work, however, if the branching rule specifically require
fast battleship, then Gangut would work.

2017年5月6日 17:03 於 "Matty Chan" <notifications@github.com> 寫道：

**Comment by Accelyte** [comment]
So if FBB is the requirement, then Gangut will work, but if fast fleet is required she will not work?
I'm just wondering since I want to level her on 5-4, but I know that slow BBs will route me A>D.

**Comment by c933103** [comment]
5-4's requirement is "fast fleet" not fast bb, so Gangut would lead you to
D there

2017年5月7日 01:20 於 "Matty Chan" <notifications@github.com> 寫道：

**Comment by LostLogia4** [comment]
Just wait until the event ends...

**Comment by ethrundr** [comment]
> Personally I think a label should be added in the KC3 library just to clarify, because not everyone is going to look it up.

KC3 panel has already provide enough data for those who care.
Gangut will kept shown as FBB there, but once she's in, fleet speed will change into *Slow*.

I don't feel quite necessary to spoonfed everything to the user, when what displayed is already giving out clear enough information. They also need to learn how to find the information they need from what we displayed.

**Comment by sinsinpub** [comment]
the fleet speed indicator never take stype into account, ofc it is slow if gangut joined... 😂

**Comment by ethrundr** [comment]
I know that, and that's why people using KC3kai need to start learning to see *that* as well, beside the ship type, when preparing a fleet required for a certain map.

**Comment by Accelyte** [comment]
> for those who care.
That's the reason why I made the thread, because I care about how it actually works.
I'm not asking for information to be spoonfed, I'm just want some clarification since the information presented is contradictory. I did my research afterwards, but you can't expect everyone else to do the same.

**Comment by c933103** [comment]
Until kancolle developers change their wording, that is how they are
present it to all 4 million players of the game. I personally don't think
KC3Kai should add any extra interpretation on it.

2017年5月17日 19:06 於 "Matty Chan" <notifications@github.com> 寫道：

**Comment by Accelyte** [comment]
To be honest, my definition of 'clarification' isn't even much. I was thinking of having the FBB indicator in a different colour and a hover tag saying that this is her ship type, not her ship speed.

**Comment by sinsinpub** [comment]
it's a simple thing: no FBB for USN, no Fast Battleship defined by Tanaka for KC game ship type (for browser ver at least). it's just misunderstanding and mistranslated by en kc community (not only kc3). just discuss to rename it like the Escort one 😂

**Comment by c933103** [comment]
​
wait api_start2 actually use senkan for both types?​


2017-05-17 20:00 GMT+08:00 sin_sin <notifications@github.com>:

**Comment by Nya-chan** [comment]
How about this, then - since the fleet can be only Fast or Slow (AFAIK?), we could pull more attention to them by changing colour (like we do with supply, HP or morale). Fast fleet is usually preferable, so it would be green, slow would be red or yellow (to prevent thinking you can't sortie or sth).

Would that work? The only way to make it more pronounced would be using the left column (like with gun fitting). Nobody is going to hover over FBB to find out what the colour means (hovering usually is very unintuitive and people will miss it, really).

**Comment by Accelyte** [comment]
That would work, I just want it so people who don't know about Gangut's interesting status will still be aware of how she works in fleet comps. With your suggestion, you could also stretch it to include turbine speed modifiers.

**Comment by Kyte** [comment]
Wouldn't it be simpler/clearer overall to rename FBB to something like escort BB, since the biggest difference between an FBB and a regular BB(V) is their ability to go in the escort fleet? (I'd suggest light BB, but then you have Iowa...)

**Comment by ethrundr** [comment]
In all honesty, if you want to make the speed change noticeable, simply add flashing animation on the fleet speed, when it changes from Fast to Slow. Or, like @Nya-chan's suggestion, notification on the left panel also work. This way, player will get notified whenever their fleet is changing speed due to adding certain ships into the fleet, not limited to Gangut only.

I still don't find why it is necessary to specifically points out only Gangut's speed status (despite she's being Slow for a FBB category), while CVLs are differed in Slow and Fast ships without decent differentation. Bet with me, lot of current players cannot correctly classified all exisiting CVLs for which are Fast and which are Slow.

There is no guarantee Gangut will be the only one with such "unique" status, so all in all, players still need to learn to see all information that has been given.

@Kyte Renaming/reclassifying them will cause confusion towards the users, as all other media are using FBB. If we want to change it, then it should be like DE/PF before: all major media must come to an agreement regarding the class change.
